src/controller/python/build-chip-wheel.py

```diff
@@ -123,6 +123,19 @@ def finalize_options(self):
     #
     # Build the chip package...
     #
+    packages=[
+            'chip',
+            'chip.configuration',
+            'chip.exceptions',
+            'chip.internal',
+            'chip.logging',
+            'chip.native',
+            'chip.tlv',
+    ]
+
+    if os.path.isdir(os.path.join(tmpDir, 'chip', 'ble')):
```

Comment:
This is wrong and breaks incremental builds. Please do not do this. You must it as an argument to the script not depend on non-existence of directories.

Reply:
Actually why does it even matter if the python modules are there or not? It is not a compiled language, we should be able to install the whole package regardless of the contents of native.